soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series

```diff
@@ -22,4 +22,11 @@ config ARCH_HAS_CUSTOM_BUSY_WAIT
 config SYS_POWER_MANAGEMENT
 	default y
 
+if SENSOR
+
+config TEMP_NRF5
```

Comment:
This is shorter and sweeter:

```
config TEMP_NRF5
   default SENSOR
```

and you can get rid of the `if SENSOR`

Reply:
Done, much cleaner method